railties/lib/rails/tasks/log.rake

```diff
@@ -7,15 +14,21 @@ namespace :log do
   end
 
   def log_files
-    if ENV['LOGS']
-      ENV['LOGS'].split(',')
-                  .map    { |file| "log/#{file.strip}.log" }
-                  .select { |file| File.exist?(file) }
-    else
+    if ENV['LOGS'] == 'all'
       FileList["log/*.log"]
+    elsif ENV['LOGS']
+      log_files_to_truncate(ENV['LOGS'])
+    else
+      log_files_to_truncate("development,test,staging,production")
```

Comment:
`staging` is not a default Rails environment. Could you remove it from here? Also from the documentation.